src/index.js

```diff
@@ -35,17 +35,7 @@ function Koop (config) {
 
   const dataset = new Dataset(this)
   const datasetController = new DatasetController(dataset)
-  bindRouteSet(routes, datasetController, this.server)
-
-  const fsRoutes = routes.concat(geoservices.routes.map(route => {
```

Comment:
are the geoservices routes still installed by default?

Reply:
yes, I'm just using the `bindRoutes` function to achieve that.